packages/server/src/db/seeds/01_users.ts

```diff
@@ -0,0 +1,23 @@
+import Knex from "knex";
+import faker from "faker";
+
+export async function seed(knex: Knex): Promise<void> {
+  await knex("users").del();
```

Comment:
It's better to move all clear steps in a separate file, for example `00_clear.ts`.